### PR TITLE
fix hash delete syntax error

### PR DESCRIPTION
**Pull request by richm**
need to get this in asap

### CODE REVIEW

**Comment by richm** [comment]
@jimmidyson we need to get this fix in asap

File: lib/fluent/plugin/filter_kubernetes_metadata.rb [hunk 1]
```diff
@@ -377,7 +377,7 @@ def start_watch
               end
               cached['kubernetes']['labels'] = labels
               if annotations.empty?
```
Comment (jimmidyson) [inline lib/fluent/plugin/filter_kubernetes_metadata.rb]: That seems reasonable - @richm can you update please?

**Comment by richm** [comment]
@portante @jimmidyson PTAL

**Comment by jimmidyson** [comment]
Will wait on green & then merge - thanks.

**Comment by jimmidyson** [comment]
Released v0.26.2.

**Pull request merged**